pkg/podman/podman.go

```diff
@@ -27,7 +27,7 @@ func NewPodmanCli(ctx context.Context) (*PodmanCli, error) {
 	}
 	version, err := cli.Version()
 	if err != nil {
-		return nil, fmt.Errorf("executable %q not found", cli.podmanCmd)
+		return nil, fmt.Errorf("executable %q not found; cause: %w", cli.podmanCmd, err)
```

Comment:
The `executable %q not found` message might be misleading, as the error is not necessarily due to that.
In this case, I'd suggest returning `err` as it is.

```suggestion
		return nil, err
```